stream_alert_cli/terraform/athena.py

```diff
@@ -33,8 +33,25 @@ def generate_athena(config):
     for refresh_type in athena_config['refresh_type']:
         data_buckets.update(set(athena_config['refresh_type'][refresh_type]))
 
+    prefix = config['global']['account']['prefix']
+    database = athena_config.get('database_name', '').strip()
```

Comment:
there was a very specific reason I did this, but I'm failing to recall what that was right now. I think I had a check in here that would remove any whitespace, since spaces are invalid in database names. I'll review and make changes